analyzers/src/SonarAnalyzer.CSharp/Rules/DoNotCallInsecureSecurityAlgorithm.cs

```diff
@@ -33,8 +33,11 @@ namespace SonarAnalyzer.Rules.CSharp
     public abstract class DoNotCallInsecureSecurityAlgorithm : SonarDiagnosticAnalyzer
     {
         internal abstract ImmutableArray<KnownType> AlgorithmTypes { get; }
+
```

Comment:
These empty lines could be reverted, as the block of abstract members is boring (and harder to read when reading the big picture of the class).